css/style.css

```diff
@@ -6,10 +6,128 @@ body {
     -webkit-font-smoothing: antialiased;
 }
 
+header { position:relative;    top:0px;    width:100%;   height:20px;  color: white; 
+    padding: 25px 0px;   display: flex;  justify-content:space-evenly; font-size: x-large;
+    align-items: center;  opacity: 0.7; z-index: 5; }
+
+    .navBar {
+        background-color: rgb(255, 255, 255);
+        margin-right: 60px;
+      }
+     
+      .navBar a {
+        color: #5a5a5a;
+        padding: 1vw 2vw;
+        text-decoration: none;
+        font-size:large;
+        text-align: center;
+        background-color: white;
+        margin-top: 2vw;
+      }
+    
+      .navBar a:hover {
+        color: orangered;
+      }
+   
+      .navBar a.active {
+        font-weight: bold;
+
+    }
+    #title-main {
```

Comment:
Don't use IDs in your CSS. You can use IDs for JavaScript, but when adding styles to HTML, target your elements using classes.